### ISSUE TITLE
[1.7.10] bug when I kill some mob shatter mod

### ISSUE THREAD

**Issue by ghost** [iChun/Shatter#38]
I'm having trouble with the shatter mod about when I kill a mob and it bugs when I'm with other mods that add multiple mobs when I try to kill them the bug shatter and crash

**Comment by talopakola** [comment]
Are you sure the mod was installed correctly?

**Comment by ghost** [comment]
@talopakola yes

**Comment by iChun** [comment]
1.7.10 is no longer supported. Even less so for a crash without a crash log.

**Comment by apiyui** [comment]
> 1.7.10 is no longer supported. Even less so for a crash without a crash log.

i think i found the problem
this mod always crash when using optifine D7 or higher versions, I'm guessing it's optifine's new shader in the D7 version that's causing the problem, use optifine D6 with GLSL shaders mod could slove this problem.
but GLSL shaders mod has some compatibility bugs with other mods, so please could you update this mod in 1.7.10 to use together with optifine shader? 
here is my crash log, forge 1.13.4.1614 with optifine E7, load only shatter mod 4.0.0 with iChunUtil 4.2.3. game crashed when i killed two sheeps after i enter this save
------------------------------------------------------------------------------------------------------------------------------------------
```
---- Minecraft Crash Report ----
// My bad.

Time: 23-9-21 上午4:26
Description: Unexpected error

java.lang.NullPointerException: Unexpected error
	at net.minecraft.client.renderer.GLAllocation.func_74523_b(SourceFile:23)
	at ichun.client.model.ModelHelper.getModelCubesCopy(ModelHelper.java:36)
	at shatter.client.model.ModelShattered.<init>(ModelShattered.java:52)
	at shatter.client.entity.EntityShattered.<init>(EntityShattered.java:35)
	at shatter.client.core.TickHandlerClient.worldTick(TickHandlerClient.java:66)
	at cpw.mods.fml.common.eventhandler.ASMEventHandler_16_TickHandlerClient_worldTick_ClientTickEvent.invoke(.dynamic)
	at cpw.mods.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:54)
	at cpw.mods.fml.common.eventhandler.EventBus.post(EventBus.java:140)
	at cpw.mods.fml.common.FMLCommonHandler.onPostClientTick(FMLCommonHandler.java:330)
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:2064)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:973)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:898)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at oolloo.jlw.Wrapper.invokeMain(Wrapper.java:60)
	at oolloo.jlw.Wrapper.main(Wrapper.java:53)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at net.minecraft.client.renderer.GLAllocation.func_74523_b(SourceFile:23)
	at ichun.client.model.ModelHelper.getModelCubesCopy(ModelHelper.java:36)
	at shatter.client.model.ModelShattered.<init>(ModelShattered.java:52)
	at shatter.client.entity.EntityShattered.<init>(EntityShattered.java:35)
	at shatter.client.core.TickHandlerClient.worldTick(TickHandlerClient.java:66)
	at cpw.mods.fml.common.eventhandler.ASMEventHandler_16_TickHandlerClient_worldTick_ClientTickEvent.invoke(.dynamic)
	at cpw.mods.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:54)
	at cpw.mods.fml.common.eventhandler.EventBus.post(EventBus.java:140)
	at cpw.mods.fml.common.FMLCommonHandler.onPostClientTick(FMLCommonHandler.java:330)

-- Affected level --
Details:
	Level name: MpServer
	All players: 1 total; [EntityClientPlayerMP['apiyui'/196, l='MpServer', x=199.13, y=68.62, z=257.73]]
	Chunk stats: MultiplayerChunkCache: 498, 507
	Level seed: 0
	Level generator: ID 00 - default, ver 1. Features enabled: false
	Level generator options: 
	Level spawn location: World: (197,64,258), Chunk: (at 5,4,2 in 12,16; contains blocks 192,0,256 to 207,255,271), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 276 game time, 276 day time
	Level dimension: 0
	Level storage version: 0x00000 - Unknown?
	Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
	Level game mode: Game mode: creative (ID 1). Hardcore: false. Cheats: false
	Forced entities: 155 total; [EntityHorse['马'/55, l='MpServer', x=123.19, y=72.00, z=266.16], EntityHorse['马'/56, l='MpServer', x=120.72, y=72.00, z=266.41], EntityHorse['马'/57, l='MpServer', x=124.06, y=72.00, z=270.09], EntityHorse['马'/58, l='MpServer', x=122.78, y=72.00, z=268.09], EntityPig['猪'/59, l='MpServer', x=131.50, y=94.00, z=312.50], EntityPig['猪'/60, l='MpServer', x=132.50, y=95.00, z=316.50], EntityPig['猪'/61, l='MpServer', x=130.28, y=95.00, z=316.44], EntityPig['猪'/62, l='MpServer', x=134.50, y=92.00, z=312.50], EntityPig['猪'/63, l='MpServer', x=148.50, y=81.00, z=318.50], EntityPig['猪'/64, l='MpServer', x=147.50, y=84.00, z=318.50], EntityPig['猪'/65, l='MpServer', x=150.50, y=80.00, z=317.50], EntityPig['猪'/66, l='MpServer', x=151.50, y=79.00, z=317.50], EntityHorse['马'/80, l='MpServer', x=171.38, y=67.00, z=213.63], EntityHorse['马'/81, l='MpServer', x=175.28, y=67.00, z=216.28], EntityHorse['马'/82, l='MpServer', x=171.50, y=68.00, z=214.50], EntityHorse['马'/83, l='MpServer', x=171.75, y=67.00, z=209.97], EntityHorse['马'/84, l='MpServer', x=175.28, y=67.00, z=208.06], EntityHorse['马'/96, l='MpServer', x=187.34, y=69.00, z=177.84], EntityHorse['马'/97, l='MpServer', x=189.50, y=68.00, z=179.50], EntityHorse['马'/98, l='MpServer', x=190.50, y=69.00, z=177.28], EntityHorse['马'/99, l='MpServer', x=196.13, y=65.00, z=200.13], EntityHorse['马'/100, l='MpServer', x=196.50, y=65.00, z=201.50], EntityHorse['驴'/101, l='MpServer', x=186.09, y=63.00, z=280.91], EntityHorse['驴'/102, l='MpServer', x=182.66, y=65.00, z=282.60], EntityPig['猪'/111, l='MpServer', x=200.50, y=68.00, z=307.50], EntityPig['猪'/112, l='MpServer', x=201.31, y=68.00, z=311.38], EntityPig['猪'/113, l='MpServer', x=198.06, y=68.00, z=312.03], EntityPig['猪'/114, l='MpServer', x=205.25, y=68.00, z=308.16], EntityPig['猪'/118, l='MpServer', x=209.50, y=64.00, z=336.50], EntityHorse['马'/119, l='MpServer', x=223.13, y=65.00, z=193.13], EntityHorse['马'/120, l='MpServer', x=223.50, y=64.00, z=194.50], EntityHorse['马'/121, l='MpServer', x=222.50, y=65.00, z=197.50], EntityHorse['马'/122, l='MpServer', x=227.50, y=67.00, z=266.72], EntityHorse['马'/123, l='MpServer', x=222.88, y=67.00, z=265.00], EntityHorse['马'/124, l='MpServer', x=222.69, y=67.00, z=263.38], EntityItem['item.tile.flower2.houstonia'/145, l='MpServer', x=234.13, y=21.13, z=249.84], EntityHorse['马'/146, l='MpServer', x=232.09, y=67.00, z=266.44], EntityHorse['马'/147, l='MpServer', x=235.00, y=67.00, z=270.28], EntityHorse['马'/148, l='MpServer', x=236.72, y=67.00, z=270.81], EntityHorse['马'/149, l='MpServer', x=236.50, y=66.00, z=274.50], EntityHorse['马'/150, l='MpServer', x=234.56, y=66.00, z=273.63], EntityHorse['马'/151, l='MpServer', x=232.72, y=66.00, z=272.06], EntityChicken['鸡'/152, l='MpServer', x=245.50, y=67.00, z=279.50], EntityChicken['鸡'/153, l='MpServer', x=243.50, y=67.00, z=276.50], EntityChicken['鸡'/154, l='MpServer', x=235.53, y=67.00, z=266.53], EntityChicken['鸡'/155, l='MpServer', x=241.50, y=67.00, z=272.50], EntityCow['牛'/156, l='MpServer', x=240.63, y=67.00, z=284.41], EntityCow['牛'/157, l='MpServer', x=235.22, y=67.00, z=290.50], EntityCow['牛'/158, l='MpServer', x=236.81, y=67.00, z=290.50], EntityCow['牛'/159, l='MpServer', x=233.50, y=67.00, z=289.50], EntityHorse['马'/164, l='MpServer', x=249.91, y=68.00, z=261.50], EntityHorse['马'/165, l='MpServer', x=251.97, y=68.00, z=261.84], EntityHorse['马'/166, l='MpServer', x=250.53, y=68.00, z=258.38], EntityHorse['马'/167, l='MpServer', x=250.91, y=69.00, z=256.91], EntityHorse['马'/168, l='MpServer', x=256.03, y=68.00, z=263.84], EntityHorse['马'/173, l='MpServer', x=268.09, y=66.00, z=194.09], EntityHorse['马'/174, l='MpServer', x=267.59, y=66.00, z=190.00], EntityHorse['马'/175, l='MpServer', x=270.66, y=66.00, z=191.16], EntityHorse['马'/176, l='MpServer', x=272.97, y=66.00, z=192.53], EntityHorse['马'/177, l='MpServer', x=271.03, y=66.00, z=194.06], EntityChicken['鸡'/178, l='MpServer', x=266.50, y=66.00, z=199.50], EntityChicken['鸡'/179, l='MpServer', x=270.50, y=66.00, z=198.50], EntityChicken['鸡'/180, l='MpServer', x=273.50, y=66.00, z=196.50], EntityChicken['鸡'/181, l='MpServer', x=271.50, y=66.00, z=197.50], EntityZombie['僵尸'/197, l='MpServer', x=157.88, y=31.00, z=333.56], EntitySkeleton['骷髅'/199, l='MpServer', x=187.50, y=37.00, z=278.50], EntityClientPlayerMP['apiyui'/196, l='MpServer', x=199.13, y=68.62, z=257.73], EntityCreeper['爬行者'/208, l='MpServer', x=162.44, y=18.00, z=259.03], EntitySquid['鱿鱼'/209, l='MpServer', x=263.97, y=59.00, z=308.50], EntityCreeper['爬行者'/214, l='MpServer', x=273.50, y=38.00, z=290.50], EntityBat['蝙蝠'/215, l='MpServer', x=232.44, y=17.24, z=243.46], EntityTrail['未知'/229, l='MpServer', x=199.13, y=68.62, z=257.73], EntityCreeper['爬行者'/230, l='MpServer', x=273.50, y=38.00, z=291.50], EntityCreeper['爬行者'/231, l='MpServer', x=249.75, y=13.00, z=236.13], EntityBat['蝙蝠'/232, l='MpServer', x=256.09, y=20.00, z=254.50], EntityBat['蝙蝠'/234, l='MpServer', x=245.91, y=20.00, z=273.06], EntityBat['蝙蝠'/235, l='MpServer', x=256.22, y=20.00, z=252.59], EntityBat['蝙蝠'/237, l='MpServer', x=217.53, y=42.44, z=187.28], EntityCreeper['爬行者'/260, l='MpServer', x=157.00, y=31.00, z=333.09], EntityCreeper['爬行者'/261, l='MpServer', x=157.50, y=31.00, z=334.50], EntityCreeper['爬行者'/262, l='MpServer', x=160.50, y=31.00, z=333.50], EntityCreeper['爬行者'/263, l='MpServer', x=160.50, y=50.00, z=207.50], EntityBat['蝙蝠'/287, l='MpServer', x=188.33, y=54.31, z=307.46], EntityCreeper['爬行者'/316, l='MpServer', x=155.50, y=50.00, z=207.50], EntityZombie['僵尸'/323, l='MpServer', x=226.50, y=43.00, z=283.50], EntitySkeleton['骷髅'/346, l='MpServer', x=183.50, y=37.00, z=275.50], EntityWitch['女巫'/348, l='MpServer', x=188.50, y=55.00, z=297.09], EntityWitch['女巫'/349, l='MpServer', x=191.50, y=55.00, z=292.50], EntityWitch['女巫'/352, l='MpServer', x=158.50, y=13.00, z=322.50], EntityCreeper['爬行者'/353, l='MpServer', x=229.50, y=43.00, z=283.50], EntityBat['蝙蝠'/355, l='MpServer', x=246.64, y=39.22, z=268.07], EntityCreeper['爬行者'/363, l='MpServer', x=229.50, y=43.00, z=282.50], EntityZombie['僵尸'/364, l='MpServer', x=228.50, y=43.00, z=281.50], EntitySkeleton['骷髅'/365, l='MpServer', x=193.50, y=27.00, z=256.50], EntityBat['蝙蝠'/366, l='MpServer', x=247.19, y=38.19, z=268.45], EntityBat['蝙蝠'/367, l='MpServer', x=229.00, y=29.00, z=188.59], EntitySkeleton['骷髅'/378, l='MpServer', x=194.50, y=27.00, z=256.50], EntitySkeleton['骷髅'/379, l='MpServer', x=192.50, y=27.00, z=257.50], EntityCreeper['爬行者'/380, l='MpServer', x=221.41, y=30.00, z=310.00], EntitySkeleton['骷髅'/382, l='MpServer', x=214.50, y=27.00, z=265.50], EntitySkeleton['骷髅'/383, l='MpServer', x=219.44, y=30.00, z=308.16], EntityBat['蝙蝠'/386, l='MpServer', x=139.56, y=23.00, z=264.28], EntitySpider['蜘蛛'/399, l='MpServer', x=187.50, y=50.00, z=207.50], EntityBat['蝙蝠'/400, l='MpServer', x=137.59, y=22.60, z=268.94], EntityBat['蝙蝠'/401, l='MpServer', x=138.00, y=23.00, z=267.91], EntityBat['蝙蝠'/402, l='MpServer', x=140.72, y=23.00, z=264.00], EntityBat['蝙蝠'/403, l='MpServer', x=186.59, y=38.00, z=266.14], EntityCreeper['爬行者'/444, l='MpServer', x=219.50, y=31.00, z=306.50], EntityCreeper['爬行者'/467, l='MpServer', x=140.50, y=23.00, z=266.50], EntitySkeleton['骷髅'/514, l='MpServer', x=171.50, y=28.00, z=242.50], EntitySpider['蜘蛛'/524, l='MpServer', x=164.16, y=55.00, z=276.56], EntitySkeleton['骷髅'/539, l='MpServer', x=192.50, y=36.00, z=284.50], EntityCreeper['爬行者'/540, l='MpServer', x=151.31, y=45.00, z=203.41], EntitySpider['蜘蛛'/564, l='MpServer', x=216.50, y=41.00, z=281.50], EntityCreeper['爬行者'/565, l='MpServer', x=139.94, y=44.00, z=204.50], EntitySkeleton['骷髅'/594, l='MpServer', x=170.22, y=32.00, z=328.41], EntityCreeper['爬行者'/595, l='MpServer', x=218.50, y=41.00, z=280.50], EntityCreeper['爬行者'/596, l='MpServer', x=217.50, y=41.00, z=278.50], EntityCreeper['爬行者'/597, l='MpServer', x=223.03, y=41.00, z=278.44], EntitySpider['蜘蛛'/598, l='MpServer', x=237.72, y=18.48, z=202.50], EntitySkeleton['骷髅'/603, l='MpServer', x=166.16, y=32.00, z=330.69], EntitySkeleton['骷髅'/604, l='MpServer', x=169.06, y=32.00, z=328.59], EntitySkeleton['骷髅'/605, l='MpServer', x=164.50, y=32.00, z=332.50], EntitySkeleton['骷髅'/606, l='MpServer', x=236.31, y=17.00, z=206.59], EntitySpider['蜘蛛'/622, l='MpServer', x=167.50, y=62.00, z=257.50], EntitySkeleton['骷髅'/623, l='MpServer', x=139.50, y=50.00, z=198.50], EntitySkeleton['骷髅'/624, l='MpServer', x=182.50, y=31.00, z=271.50], EntitySpider['蜘蛛'/644, l='MpServer', x=255.59, y=13.00, z=231.59], EntityZombie['僵尸'/645, l='MpServer', x=183.74, y=60.00, z=263.82], EntityZombie['僵尸'/646, l='MpServer', x=252.50, y=21.00, z=217.50], EntityCreeper['爬行者'/648, l='MpServer', x=159.50, y=13.00, z=324.50], EntityCreeper['爬行者'/692, l='MpServer', x=205.66, y=27.00, z=247.59], EntitySkeleton['骷髅'/727, l='MpServer', x=209.06, y=30.00, z=241.53], EntitySpider['蜘蛛'/789, l='MpServer', x=225.50, y=21.00, z=253.50], EntityZombie['僵尸'/804, l='MpServer', x=277.50, y=36.00, z=259.50], EntityCreeper['爬行者'/825, l='MpServer', x=166.50, y=26.00, z=204.50], EntityZombie['僵尸'/887, l='MpServer', x=250.50, y=21.00, z=228.50], EntitySkeleton['骷髅'/888, l='MpServer', x=240.50, y=20.00, z=193.50], EntityCreeper['爬行者'/978, l='MpServer', x=257.50, y=19.00, z=251.50], EntitySkeleton['骷髅'/997, l='MpServer', x=257.50, y=19.00, z=252.50], EntityZombie['僵尸'/998, l='MpServer', x=163.50, y=13.00, z=269.50], EntityZombie['僵尸'/1004, l='MpServer', x=230.41, y=15.00, z=206.03], EntityCreeper['爬行者'/1094, l='MpServer', x=150.00, y=20.00, z=240.59], EntityCreeper['爬行者'/1109, l='MpServer', x=224.50, y=25.00, z=253.50], EntityZombie['僵尸'/1110, l='MpServer', x=229.50, y=17.00, z=270.94], EntityCreeper['爬行者'/1163, l='MpServer', x=166.50, y=26.00, z=208.50], EntityItem['item.item.seeds'/1284, l='MpServer', x=212.06, y=63.13, z=233.91], EntitySheep['羊'/3743, l='MpServer', x=197.08, y=67.39, z=259.73], EntityItem['item.item.string'/3747, l='MpServer', x=209.38, y=27.13, z=269.00], EntityShattered['未知'/3749, l='MpServer', x=210.50, y=26.45, z=269.50], EntityItem['item.tile.cloth.white'/3750, l='MpServer', x=199.69, y=67.13, z=259.69], EntityShattered['未知'/3752, l='MpServer', x=198.55, y=66.98, z=260.60], EntityXPOrb['经验球'/3758, l='MpServer', x=199.24, y=67.35, z=261.18], EntityXPOrb['经验球'/3759, l='MpServer', x=198.40, y=68.12, z=260.16], EntityItem['item.tile.cloth.white'/3762, l='MpServer', x=196.90, y=67.28, z=259.67]]
	Retry entities: 0 total; []
	Server brand: fml,forge
	Server type: Integrated singleplayer server
Stacktrace:
	at net.minecraft.client.multiplayer.WorldClient.func_72914_a(WorldClient.java:425)
	at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:2444)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:927)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at oolloo.jlw.Wrapper.invokeMain(Wrapper.java:60)
	at oolloo.jlw.Wrapper.main(Wrapper.java:53)

-- System Details --
Details:
	Minecraft Version: 1.7.10
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_281, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 343231472 bytes (327 MB) / 616562688 bytes (588 MB) up to 6442450944 bytes (6144 MB)
	JVM Flags: 6 total; -XX:+UseG1GC -XX:-UseAdaptiveSizePolicy -XX:-OmitStackTraceInFastThrow -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmn256m -Xmx6144m
	AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	IntCache: cache: 0, tcache: 0, allocated: 13, tallocated: 95
	FML: MCP v9.05 FML v7.10.99.99 Minecraft Forge 10.13.4.1614 Optifine OptiFine_1.7.10_HD_U_E7 5 mods loaded, 5 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJAAAA	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJAAAA	FML{7.10.99.99} [Forge Mod Loader] (forge-1.7.10-10.13.4.1614-1.7.10.jar) 
	UCHIJAAAA	Forge{10.13.4.1614} [Minecraft Forge] (forge-1.7.10-10.13.4.1614-1.7.10.jar) 
	UCHIJAAAA	iChunUtil{4.2.3} [iChunUtil] (iChunUtil-4.2.3.jar) 
	UCHIJAAAA	Shatter{4.0.0} [Shatter] (Shatter-4.0.0.jar) 
	GL info: ' Vendor: 'NVIDIA Corporation' Version: '4.6.0 NVIDIA 537.34' Renderer: 'NVIDIA GeForce RTX 2070/PCIe/SSE2'
	Launched Version: 1.7.10-Forge_10.13.4.1614-OptiFine_E7
	LWJGL: 2.9.1
	OpenGL: NVIDIA GeForce RTX 2070/PCIe/SSE2 GL version 4.6.0 NVIDIA 537.34, NVIDIA Corporation
	GL Caps: Using GL 1.3 multitexturing.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Anisotropic filtering is supported and maximum anisotropy is 16.
Shaders are available because OpenGL 2.1 is supported.

	Is Modded: Definitely; Client brand changed to 'fml,forge'
	Type: Client (map_client.txt)
	Resource Packs: [PCL2 Skin.zip]
	Current Language: 简体中文 (中国)
	Profiler Position: N/A (disabled)
	Vec3 Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	Anisotropic Filtering: Off (1)
	OptiFine Version: OptiFine_1.7.10_HD_U_E7
	Render Distance Chunks: 12
	Mipmaps: 4
	Anisotropic Filtering: 1
	Antialiasing: 0
	Multitexture: false
	Shaders: null
	OpenGlVersion: 4.6.0 NVIDIA 537.34
	OpenGlRenderer: NVIDIA GeForce RTX 2070/PCIe/SSE2
	OpenGlVendor: NVIDIA Corporation
	CpuCount: 12
```
